packages/teams-js/src/private/store.ts

```diff
@@ -39,14 +40,25 @@ export enum StoreDialogType {
   AppDetail = 'appdetail',
 }
 
+/**
+ * @beta
+ * @hidden
+ * Interface of store dialog size
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export interface StoreSizeInfo {
```

Comment:
Because this is optional you should document what happens if the size is omitted (for example, you could say something like "the host will choose an appropriate size for you".

Reply:
updated